thunder/__init__.py

```diff
@@ -1014,6 +1015,14 @@ def last_compile_options(fn: Callable, /) -> None:
         print(f"\t{option}")
 
 
+def get_auto_registered_torch_op_names(fn: Callable, /) -> set[str] | None:
+    """Returns a set of auto-registered Torch operator names present in the given JIT-compiled function."""
+    op_names = set()
```

Comment:
```
trc = last_traces(fn)[0]
return {bsym.sym.id for bsym in trc.bound_symbols if has_tags(bsym, {prims.OpTags.AUTO_REGISTERED}}
```

Use a set comprehension directly, like this. It's OK to return an empty set to indicate that there were no auto registered operations